File: URI_1173.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int v,j=1;
    	v = int.Parse(Console.ReadLine());
    	
        int []sum = new int[10];
        
        for (int i = 0; i < sum.Length ; i++) {
            sum[i] = v * j ;
            j+=j ;
        }
        for (int i = 0; i < sum.Length ; i++) {
            Console.WriteLine("N[" + i + "] = " + sum[i]);
        }

    }

}

File: URI_1173.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
        int v,j=1;
    	cin>>v;
    	
        int sum[10];
        
        for (int i = 0; i < 10 ; i++) {
            sum[i] = v * j ;
            j+=j ;
        }
        for (int i = 0; i < 10 ; i++) {
            cout<<"N["<<i<<"] = "<<sum[i]<<endl;
        }
 
    return 0;
}
